lua/refactoring/tests/refactor_spec.lua

```diff
@@ -201,13 +206,21 @@ describe("Refactoring", function()
             )
 
             Config.get():reset()
-            -- TODO: How to get this dynamically?
+
+            -- Needed for indenting tests with certain languages
+            if no_expand_tab[filename_extension] then
```

Comment:
this is dangerous, but luckily its only set to go, which the formatter requires it... but i would be very hesitant to change any options like this.

Reply:
For now I think it's okay because it's for testing purposes - for me at least the functionality works when done manually but not in the tests (unless this snippet is there).